docs/recipes/install/common/openeuler_repos.tex

```diff
@@ -8,17 +8,17 @@
 
 \begin{itemize*}
 \item OS
-  (e.g. \href{http://repo.openeuler.org/openeuler/openEuler-22.03-LTS/OS/}
-             {\color{blue}{http://repo.openeuler.org/openeuler/openEuler-22.03-LTS/OS/}} )
+  (e.g. \href{http://repo.openeuler.org/openEuler-22.03-LTS/OS/}
+              {\color{blue}{http://repo.openeuler.org/openEuler-22.03-LTS/OS/}} )
```

Comment:
There is a single space added which seems unnecessary

Reply:
Fixed! Thanks!